src/ds18b20.toit

```diff
@@ -142,12 +149,174 @@ class Ds18b20:
   */
   is_parasitic -> bool:
     if is_closed: throw "CLOSED"
+    if is_parasitic_ != null: return is_parasitic_
     if not bus_.reset:
       throw "NO DEVICE FOUND"
     select_self_
     bus_.write_byte READ_POWER_SUPPLY_
     result := bus_.read_bits 1
-    return result == 0
+    is_parasitic_ = result == 0
+    return is_parasitic_
+
+  /**
+  Reads the scratchpad and returns the raw bytes.
+
+  The scratchpad is the DS18B20's memory. It contains the temperature, the
+    alarm temperatures, the configuration register, and the CRC. Unless
+    committed to EEPROM, the scratchpad is volatile and is reset to the
+    EEPROM values on power up.
+
+  If $check_crc is true, then verifies the CRC of the scratchpad (the last byte).
+
+  The result contains the following bytes:
+  - 0: temperature LSB
```

Comment:
I think these should be named constants.
HIGH_ALARM_READ_INDEX ::= 2
HIGH_ALARM_WRITE_INDEX ::= 0

Reply:
done.